static/src/stylesheets/module/experiments/_acquisitions-epic-testimonials.scss

```diff
@@ -0,0 +1,89 @@
+// CSS used for the Epic testimonials test
+
+// Common classes
+
+.epic__testimonial__container {
+    margin-top: $gs-gutter;
+    margin-bottom: $gs-gutter;
+}
+
+.epic__testimonial__quote {
+    position: relative;
+}
+
+.epic__testimonial__quote span {
+    position: absolute;
+}
+
+.epic__testimonial__quote svg {
+    fill: #ffbb00;
+    stroke: #ffbb00;
+    width: 1.5 * $gs-gutter;
+    height: 1.5 * $gs-gutter;
+}
+
+.epic__testimonial__text {
+    margin-bottom: 0;
+    font-family: $f-sans-serif-text;
+    font-style: italic;
+    padding-left: 2 * $gs-gutter;
+}
+
+.epic__testimonial__name {
+    font-family: $f-sans-serif-text;
+    font-style: normal;
+}
+
+
+// Variant 1 - control
+
+// Variant 2 - prominent
+
+// Variant 3 - subtle
+
+.epic__testimonial__quote--subtle svg {
+    fill: #bdbdbd;
+    stroke: #bdbdbd;
```

Comment:
`$neutral-3` Could you also double-check whether we really need both, `fill` and `stroke`? Let me know if you need help understanding the difference :)